data/distro.yml

```diff
@@ -610,7 +610,34 @@
         <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
       </li>
     </ol>
-
+    
+- name: "Aylinux"
+  logo: "aylinux.svg"
+  info: >
+    <ol class="distrotut">
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>To install Flatpak, run the following in a terminal:</p>
+        <pre><code>
+          <span class="unselectable">$</span> su
+          <span class="unselectable">#</span> paket kur flatpak
+        </code></pre>
+      </li>
+      <li>
+        <h2>Add the Flathub repository</h2>
+        <p>Flathub is the best place to get Flatpak apps. To enable it, run:</p>
+        <pre><code>
+          <span class="unselectable">$</span> su
```

Comment:
Does Aylinux support Polkit for privilege elevation? If so, this is not needed. If not, installation of new apps will probably require root permissions as well.